index_deskop.html

```diff
@@ -0,0 +1,332 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="desktop.css"/>
+    <link rel="stylesheet" href="handheld.css">
+    <title>Desktop Version</title>
+  </head>
+  <body>
+    <div class="background">
+      <nav class=" flex-container-navigation">
+        <a class="flex-item flex-item-1 p-1-nav logo" href="#">My Logo</a>
+        <ul class="flex-item flex-item-2 top_nav_ul">
+          <li> <a href="#">Portfolio </a> </li>
+          <li> <a href="#">About </a> </li>
+          <li> <a href="#">Contact </a> </li>
+        </ul>
+        <a href="javascript:void(0)" class="not_show_bars p-1-nav" onclick="myFunction()"><i class="fa fa-bars"></i></a>
+      </nav>
+      <div class="content">
+      <main>
+        <h1>
+          I’m Waqas <br />
+          Glad to see you!
+        </h1>
+        <p class="description">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <h4 class="connect">LET’S CONNECT</h4>
+        <div>
+          <ul class="on_row">
```

Comment:
- [ ] Again here as requested by the previous reviewer, the social media icons are still not being displayed except the last one which is the `M` icon. It would be nice to fix it.